src/js/card.js

```diff
@@ -0,0 +1,151 @@
+import { playAudio } from './function'
+
+export class Card {
+  constructor(id, cardData) {
+    this.id = id
+    this.cardData = cardData
+  }
+
+  buildCardContainer(selector, className) {
+    const block = document.querySelector(selector)
+    const card = document.createElement('div')
+    const cover = document.createElement('div')
+    const infoContainer = document.createElement('div')
+    const cardTitle = document.createElement('h2')
+    const indicator = document.createElement('div')
+
+    card.classList.add(className)
+    cover.classList.add('card__cover')
+    infoContainer.classList.add('card__info_block')
+    cardTitle.classList.add('info_block__title')
+    indicator.classList.add('info_block__indicator')
+
+    card.setAttribute('id', this.id)
+
+    block.appendChild(card)
+
+    card.appendChild(cover)
+
+    card.appendChild(infoContainer)
+    infoContainer.appendChild(cardTitle)
+    infoContainer.appendChild(indicator)
```

Comment:
Please move to separate method common logic with creating new HTML element

Reply:
+++